src/command.c

```diff
@@ -670,6 +670,14 @@ static bool cmd_shutdown_bmda(target_s *t, int argc, const char **argv)
 }
 #endif
 
+/*
+ heapinfo can be used to quickly test a system with different heap and stack values, to see how much heap and stack is needed.
+ - user sets up values for heap and stack using "mon heapinfo"
+ - when the target boots, crt0.S does a heapinfo semihosting call to get these values for heap and stack.
+   See newlib/libc/sys/arm/crt0.S "Issue Angel SWI to read stack info"
+ - if the system crashes increase heap or stack
+ */
```

Comment:
Please make this comment match the code base multi-line comment style:
```
/*
 * Line 1
 * Line 2
 * - Bullet 1
 * - Bullet 2
 */
```

Please also use proper start-of-bullet and sentence capitalisation.

Looks good otherwise.